File: 5 - Coding Interview Workshops/Week7/CodeFiles/student.py
class Student:
    def __init__(self, name, age, grade):
        self.name = name
        self.age = age
        self.grade = grade

    def sayMyName (self):
        print("Hi, my name is " + self.name)

zahra = Student("Zahra", 23, 12)
print(zahra.name)

File: 5 - Coding Interview Workshops/Week07/CodeFiles/JavaScript/student.js
class Student {
    constructor (name, age, grade) {
        this.name = name;
        this.age = age;
        this.grade = grade;
    }

    sayMyName () {
        console.log(`Hi, my name is ${this.name}`);
    }
}

let zahra = new Student("Zahra", 23, 12);
console.log(zahra.name);

